ooniprobe/View/RunTest/TestOverviewViewController.m

```diff
@@ -37,19 +38,15 @@ - (void)viewDidLoad {
         [[UIApplication sharedApplication] openURL:url];
     }];
     [self.runButton setTitle:[NSString stringWithFormat:@"%@", NSLocalizedString(@"Dashboard.Overview.Run", nil)] forState:UIControlStateNormal];
-    if ([testSuite.name isEqualToString:@"websites"])
+    if ([[descriptor performSelector:@selector(name)] isEqualToString:@"websites"])
         [self.websitesButton setTitle:[NSString stringWithFormat:@"%@", NSLocalizedString(@"Dashboard.Overview.ChooseWebsites", nil)] forState:UIControlStateNormal];
     else
         [self.websitesButton setHidden:YES];
 
     [self reloadLastMeasurement];
-    [self.testImage setImage:[UIImage imageNamed:[NSString stringWithFormat:@"%@", testSuite.name]]];
-    defaultColor = [TestUtility getBackgroundColorForTest:testSuite.name];
+    [self.testImage setImage:[UIImage imageNamed:[NSString stringWithFormat:@"%@", [descriptor performSelector:@selector(name)]]]];
+    defaultColor = [TestUtility getBackgroundColorForTest:[descriptor performSelector:@selector(name)]];
     [self.runButton setTitleColor:defaultColor forState:UIControlStateNormal];
-    if (testSuite.getTestList.count <= 0) {
-        self.runButton.userInteractionEnabled = NO;
-        [self.runButton setTitleColor:[UIColor colorNamed:@"color_gray3"] forState:UIControlStateNormal];
-    }
```

Comment:
Can you elaborate on the implication of removing this block of code?

Reply:
This code was used to check if at least one test is enabled, which would then disabled the runButton.

Since there is going to be no run button anymore, no need to have it.

Reply:
Gotcha!